### PR TITLE
Added in "normal" severity option to documentation

### PR DESCRIPTION
**Pull request by jackjohnsonuk**

### CODE REVIEW

File: bignumber-card/README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Severity object
 | red | number | **Required** | Value from which to start red color
 | green | number | **Required** | Value from which to start green color
 | amber | number | **Required** | Value from which to start amber color
+| normal | number | **Required** | Value from which is classed as a normal color N.B. this cannot be used with red/green/amber
```
Comment (ciotlosm) [inline bignumber-card/README.md]: This should not be required. There should be a note that you should always have exactly three different severities.

Example: (amber, red, normal) or (green, normal, red) etc.

Comment (jackjohnsonuk) [reply bignumber-card/README.md]: would green, and normal not be the same thing? I assumed normal would give it a green band and then when it wondered off "normal" it changed to a red for example. So does normal display as a white background then?

Comment (ciotlosm) [reply bignumber-card/README.md]: Blue (primary color of hass actually)

**Comment by jackjohnsonuk** [comment]
hopefully I've got a better understanding of "normal" now :)

**Pull request merged**